footer.php

```diff
@@ -13,6 +13,7 @@
 		</div><!-- .container -->
 	</div><!-- #content -->
 
+	<div class="clear"></div>
```

Comment:
Rather than using `clear` divs, we should be using `clearfix` elements. This is especially true now that we only need to support fairly modern browsers. There is a clearfix mixin that we can use. In this case, `#content` would be the element we clearfix.